frontend/src/app/(routes)/transfers/SendPage.tsx

```diff
@@ -0,0 +1,321 @@
+                  message: 'Please enter a valid number',
+                },
+                validate: {
+                  invalid: (value) =>
+                    Number(value) > 0 || "Amount can't be zero",
+                  insufficient: (value) =>
+                    Number(selectedAsset?.balance) >
+                      Number(value) +
+                        Number(
+                          SEND_TX_FEE / 10 ** (selectedAsset?.decimals || 0)
+                        ) || 'Insufficient funds',
+                },
+              }}
+              render={({ field }) => (
+                <TextField
+                  className="bg-[#FFFFFF0D] rounded-2xl"
+                  {...field}
+                  required
+                  fullWidth
+                  size="small"
+                  placeholder="Enter Amount here"
+                  sx={{
+                    '& .MuiTypography-body1': {
+                      color: 'white',
+                      fontSize: '12px',
+                      fontWeight: 200,
+                    },
+                    '& .MuiOutlinedInput-notchedOutline': {
+                      border: 'none',
+                    },
+                  }}
+                  InputProps={{
+                    endAdornment: (
+                      <InputAdornment position="start">
+                        {selectedAsset?.displayDenom}
+                      </InputAdornment>
+                    ),
+                    sx: {
+                      input: {
+                        color: 'white',
+                        fontSize: '14px',
+                        padding: 2,
+                      },
+                    },
+                  }}
+                  error={!!errors.amount}
+                />
+              )}
+            />
+
+            <div
+              className={`text-red-500 text-xs px-4 opacity-95 mt-1 min-h-[24px]`}
+            >
+              {!sortedAssets.length
+                ? 'No Assets Found.'
+                : !selectedAsset
+                  ? 'Please select an Asset'
+                  : errors.amount
+                    ? errors.amount?.message
+                    : ''}
+            </div>
+            <div className="min-h-[96px]">
+              <div className="flex items-center gap-2  mb-2">
+                <div className="flex items-center text-sm not-italic font-normal leading-[normal]">
+                  Memo
+                </div>
+                <Image
+                  onClick={() => setOpenMemo((openMemo) => !openMemo)}
+                  src="/drop-down-icon.svg"
+                  className='cursor-pointer'
+                  width={16}
+                  height={16}
+                  alt=""
+                />
+              </div>
+              {openMemo ? (
+                <Controller
+                  name="memo"
+                  control={control}
+                  render={({ field }) => (
+                    <TextField
```

Comment:
I suggest creating a reusable text field component and passing necessary data as props to improve form data management.